app/assets/stylesheets/config/_colors.scss

```diff
@@ -6,7 +6,8 @@ $orange: #F5A623;
 $green: #61C77E;
 
 // secondary colors
-$font_color: #333745;
+$font_color: black;
```

Comment:
Color was correct, but it was not applied. The font was blue not #333745. It is supposed to be #33745

Reply:
ok, done